sensu/templates/client.json

```diff
@@ -1,9 +1,14 @@
 {% set roles = salt['grains.get']('roles', []) -%}
+{% set custom_prefix = salt['pillar.get']('collectd:prefix', '') %}
+{% if custom_prefix[:8] == 'metrics.' %}
+{% set custom_prefix = custom_prefix[8:] %}
```

Comment:
It would be *really* nice if you could explain this logic in the commit message. I *think* i know whats going on here, but it's making my head hurt.

Also, why do we need a custom_prefix in the first place, when we're just taking the pillar value from collectd?